src/app/@shared/near-me-items/near-me-items.component.html

```diff
@@ -13,13 +13,13 @@ <h3 *ngIf="waitLocalVendors.waiting == false && waitLocalVendors.waitedForTooLon
         </h3>
         <!-- NO VENDORS FOUND -->
         <ng-template #elseBlock>
-            <h3 *ngIf="!topVendorsNearMe.length && waitLocalVendors.waiting === false">{{ "NO_VENDORS_NEAR" | translate }}</h3>
+            <p class="noItemsText" *ngIf="!topVendorsNearMe.length && waitLocalVendors.waiting === false">😞 {{ "NO_VENDORS_NEAR" | translate }}</p>
         </ng-template>
         <!-- VENDORS FOUND IMMEDIATELY -->
         <app-list-vendors
             *ngIf="waitLocalVendors.waiting === false && waitLocalVendors.waitedForTooLong === false"
             [vendorsWithListings$]="topVendorsNearMe$"
-            [title]="'Vendors near your location'"
+            [title]="'📍 Vendors near your location'"
```

Comment:
<3 the emojis, maybe we should use Translation to this title:
```
[title]="VENDORS_NEAR_LOCATION | translate"
```
also add to translations.
I know it's not your implementation, but we might just fix it sooner :)

Reply:
I get a syntax error of that